src/tools/quemap/lightgrid.c

```diff
@@ -453,7 +454,18 @@ static void FogLuxel(GArray *fogs, luxel_t *l, float scale) {
 				break;
 		}
 
-		intensity *= fog->density + RandomRangef(-fog->noise, fog->noise);
+		// TODO: Should the divisors below be lg->w instead of MAX_BSP_LIGHTGRID_WIDTH?
```

Comment:
@Paril This is my one remaining question on this one before we merge in. This looks really nice!